var/spack/repos/builtin/packages/pangolin/package.py

```diff
@@ -17,6 +17,7 @@ class Pangolin(CMakePackage):
     depends_on('cmake@2.8.12:', type='build')
     depends_on('gl')
     depends_on('glew')
+    depends_on('glu', type='link')
```

Comment:
Is this a dependency for all versions or only the latest?

Reply:
Resolving this since, as of now, the package tracks only tip of the develop branch